docs/site/index/language-support-levels

```diff
@@ -0,0 +1,129 @@
+---
```

Comment:
There appears to be something off in this file. There is a warning symbol at the end, and I can't view as HTML

Reply:
Oops, it wasn't a markdown file. Should be good now.